example/trace/mock-server/main.go

```diff
@@ -0,0 +1,69 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"time"
+
+	"google.golang.org/api/option"
+
+	"github.com/googleinterns/cloud-operations-api-mock/cloudmock"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+
+	cloudtrace "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace"
+)
+
+func main() {
+	// create new mock server
+	mock := cloudmock.NewCloudMock()
```

Comment:
IMO it would be more useful for the end users if we documented a way to start a standalone mock server, and setting up their pipeline to forward spans to it.
e.g.
> "For local testing, you can start a mock server as described in https://github.com/googleinterns/cloud-operations-api-mock#stand-alone-server-mode and set the host/port in the client as follows: {code snippet}

We don't expect end users to write code for mocking inside of their applications.

@james-bebbington what do you think?

Reply:
I was just writing up a very similar comment - I totally agree with your suggestion